arches_her/pkg/graphs/resource_models/Digital Object.json

```diff
@@ -1287,7 +1287,7 @@
                 {
                     "card_id": "7db89892-8490-11ea-8777-f875a44e0e11",
                     "config": {
-                        "acceptedFiles": "",
+                        "acceptedFiles": "Any File Type",
```

Comment:
This causes an error that likely means the value is both the label and also a setting.  This should be reverted and a change submitted to core so that an empty string (used by the dropzone component to mean any file type) is replaced in the widget by a translation of "Any File Type"

![image](https://github.com/archesproject/arches-her/assets/25295526/8efe7531-7dbd-4945-90c8-43e2b6a3da1d)

![image](https://github.com/archesproject/arches-her/assets/25295526/679a5d0c-7793-4ed4-a83c-9895cade6567)